README.md

```diff
@@ -24,7 +24,7 @@ coverage-helper = "0.1"
 And add this to your crate root (`lib.rs` or `main.rs`):
 
 ```rust
-#![cfg_attr(coverage_nightly, feature(no_coverage))]
+#![cfg_attr(coverage_nightly, feature(coverage))]
```

Comment:
The feature is/will be called `coverage_attribute` (since the name `coverage` was deemed, correctly, to be confusing).

Ditto in the test's lib.rs.

Reply:
Oh, good catch. Thanks.

Reply:
I was just checking that `llvm-cov` didn't have the same issue, and you fixed it!